Mail/Template.php

```diff
@@ -205,10 +212,18 @@ public function loadDefault($templateId)
     public function getProcessedTemplate(array $variables = [])
```

Comment:
@sofvlad please check all core methods from M2.3.5 and compare them to the overridden ones in Sailthru extension (for example method getProcessedTemplate()). If any new code was added to core class - migrate it (copy) to our class. Do this for all preference classes and make sure __construct() methods are also compatible with core classes (if we override __construct() methods, for ex. see Sailthru/Mail/Transport.php __construct() method